cloudbuild/cloudbuild.yaml

```diff
@@ -0,0 +1,6 @@
+steps:
```

Comment:
Pretty sure you can't do this from inside cloud build easily. Docker in docker is a headache. Doing it manually is fine for now IMO!